src/Servers/Kestrel/Core/src/Internal/Http3/Frames/Http3ErrorCode.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3
+{
+    internal enum Http3ErrorCode : uint
+    {
+        /*
+         * https://quicwg.org/base-drafts/draft-ietf-quic-http.html#http-error-codes
+            HTTP_NO_ERROR (0x100):
+            No error. This is used when the connection or stream needs to be closed, but there is no error to signal.
```

Comment:
Can you merge these together so the comments are doc comments on the individual values?